### PR TITLE
Image Sequences

### PR DESCRIPTION
**Pull request by anderspollack**
- Prevent hidden slide elements from blcoking current slide interactions
- Refactor handleSequenceNavigation
- Add support for linking to a specific index of an image sequence with `annoref`

### CODE REVIEW

File: content/_assets/javascript/application/canvas-panel.js [hunk 1]
```diff
@@ -23,12 +23,17 @@ const annotationData = (input) => {
 const getServiceId = (element) => {
   const canvasPanel = element.querySelector('canvas-panel')
   const imageService = element.querySelector('image-service')
+  const imageSequence = element.querySelector('image-sequence')
```
Comment (mphstudios) [inline content/_assets/javascript/application/canvas-panel.js:26]:
```suggestion
  const imageSequence = element.querySelector('image-sequence')
  const imageService = element.querySelector('image-service')
```

Comment (mphstudios) [inline content/_assets/javascript/application/canvas-panel.js]: @anderspollack we might make a method to abstract getting the `id` for these different elements, sketch below, though we should do so in a separate pull-request and perhaps when we refactor figures?
```
const serviceElement = element.querySelector('canvas-panel', 'image-sequence', 'image-service')
const service = new Service(serviceElement)
return service.id
```

**Review by mphstudios — APPROVED** [review]

File: content/_assets/javascript/application/canvas-panel.js (hunk 1 above)
Comment (anderspollack) [reply content/_assets/javascript/application/canvas-panel.js]: @mphstudios I think it makes sense to do that when we refactor all figures. If we can reasonably refactor figure image elements to use webc this file will be obsolete

**Comment by anderspollack** [comment]
@geealbers these are some required changes to support 360 images. It shouldn't affect anything in current quire but I wanted to give you a head's up before I merge this in in case something were to come up.

**Comment by geealbers** [comment]
Thanks @anderspollack. Are there things this merge _could_ adversely affect? Are there particular things we should go test afterward?

**Comment by anderspollack** [comment]
@geealbers canvas panel region annoref links, and general application JS errors showing up in the browser console

**Review by geealbers — APPROVED** [review]

**Pull request merged**